### ISSUE TITLE
Javadocs on Input interface for accelerometer are wrong

### ISSUE THREAD

**Issue by spectacularbob** [libgdx/libgdx#4652]
In the source for the Input interface

/** @return The rate of rotation around the x axis. (rad/s) */
public float getAccelerometerX ();

/** @return The rate of rotation around the y axis. (rad/s) */
public float getAccelerometerY ();

/** @return The rate of rotation around the z axis. (rad/s) */
public float getAccelerometerZ ();

public float getGyroscopeX ();

public float getGyroscopeY ();

public float getGyroscopeZ ();

In the AndroidInput.java implementation of this interface, getAccelerometer returns values from the event Sensor.TYPE_ACCELEROMETER which returns acceleration in m/s^2 on an axis, not rotation.

In short, the javadocs should be moved to the getGyroscope functions to be correct
